packages/nameguard-react/src/components/Report/CheckResultCodeIcon.tsx

```diff
@@ -77,17 +83,19 @@ function icon(
  * @param {CheckResultCode} code - The CheckResultCode status to display.
  * @param {boolean} isInteractive - Wether to display a Tooltip and a color effect on icon hover or not.
  */
-export function ResultCodeIcon({
+export function CheckResultCodeIcon({
   code,
+  onClick = () => {},
   isInteractive = false,
   className = "",
 }: {
+  onClick?: () => void;
   code: CheckResultCode | Rating;
```

Comment:
Let's please ensure we remove any code that doesn't strictly separate `CheckResultCode` from `Rating`. These are never to be treated like they are the same thing.